src/components/Dashboard/FilingHistoryList/StaffNotation.vue

```diff
@@ -0,0 +1,177 @@
+                            <v-list-item-title
+                                class="add-detail-comment-item"
+                            >
+                                <span class="app-action">Add Court Order</span>
+                            </v-list-item-title>
+                        </v-list-item>
+                    </v-list-item-group>
+                </v-list>
+            </v-menu>
+        </div>
+    </div>
+</template>
+<script lang="ts">
+import { Component, Prop, Emit, Vue } from 'vue-property-decorator'
+
+// Dialog
+import { AddStaffNotationDialog } from '@/components/dialogs'
+
+@Component({
+  components: { AddStaffNotationDialog }
+})
+export default class StaffNotation extends Vue {
+  private isAddingRegistrarsNotation = false
+  private isAddingRegistrarsOrder = false
+  private isAddingCourtOrder = false
+  private expand = false
+
+  /** Prop for the scrollbar offset to be added. */
+  @Prop() private addScrollbarOffset: string
+
+  showRegistrarsNotationDialog (): void {
+    this.isAddingRegistrarsNotation = true
+  }
+  hideRegistrarsNotationDialog (needReload: boolean): void {
+    this.isAddingRegistrarsNotation = false
+    this.close(needReload)
+  }
+  showRegistrarsOrderDialog (): void {
+    this.isAddingRegistrarsOrder = true
+  }
+  hideRegistrarsOrderDialog (needReload: boolean): void {
+    this.isAddingRegistrarsOrder = false
+    this.close(needReload)
+  }
+  showCourtOrderDialog (): void {
+    this.isAddingCourtOrder = true
+  }
+  hideCourtOrderDialog (needReload: boolean): void {
+    this.isAddingCourtOrder = false
+    this.close(needReload)
+  }
+
+  @Emit('close')
+  private close (needReload: boolean): void {
+    // Intentionally empty
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "@/assets/styles/theme.scss";
+#add-staff-filing-label {
+    padding-right: 0.725rem;
+    font-size: 0.875rem;
+    color: $app-blue;
+}
+
+#staff-notation {
+    padding-right: 1.5rem;
+}
+
+.add-scrollbar-offset {
+    overflow-y: scroll;
+    scrollbar-color: transparent transparent; // FireFox uses this property
+
+    // Webkit browsers use ::-webkit-scrollbar pseudo element
+    &::-webkit-scrollbar {
+        background: transparent;
+    }
+}
+#staff-notation > div.filing-item__actions > button {
+    font-size: 0.875rem;
+    color: $app-blue;
+}
+#add-staff-filing-icon {
+    font-size: 1.2rem;
+    padding: 0.2rem;
+    margin-bottom: 0.2rem;
+    color: $app-blue;
+}
+
+.filing-item__actions {
+  display: inline-block;
+  margin-right: 0.5rem;
+
+  .expand-btn {
+    letter-spacing: -0.01rem;
+    font-size: 0.875rem;
+    font-weight: 700;
+  }
+
+  // make menu button slightly smaller
+  .menu-btn {
+    height: unset !important;
+    min-width: unset !important;
+    padding: 0.25rem !important;
+    color: $app-blue
+  }
+}
+#app > div.v-menu__content {
+    margin: 0.625rem 0 0 0;
+}
+.v-btn.active .v-icon {
+    transform: rotate(-180deg);
+  }
```

Comment:
Fix indent.

That's a slick way to flip the down arrow :)

Reply:
That would keep the arrow animation like in a select: https://vuetifyjs.com/en/components/selects/

Reply:
Yes, very cool. I think in other places we toggle the v-icon between up-arrow and down-arrow. This is sexier :)

Reply:
Agreed. 😄